cpp/ps0001/ps0001_sol.cpp

```diff
@@ -0,0 +1,25 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+  // prompting user for 3 numbers to add 1/2
+  short num1;
+  cout<<"please enter first number: "<<endl;
+  cin>>num1;
+  
+  short num2;
+  cout<<"please enter second number: "<<endl;
+  cin>>num2; 
+  
+  short num3;
+  cout<<"please enter third number: "<<endl;
+  cin>>num3;
+
+  // calculating & outputting the result 3/3
+  cout<<A<<"+"<<B<<"+"<<C<<"="<<A+B+C<<endl;
+  
+  return 0;
+}
+// good work 👍 6/10
```

Comment:
- [ ] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [ ] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code